### ISSUE TITLE
GSE119561 (organoids): D25 organoid, D18 organoid

### ISSUE THREAD

**Issue by ami-day** [ebi-ait/hca-ebi-wrangler-central#377]
**Dataset/group this task is for:**

Barbara Treitlen, Melissa Little

**Wrangler responsible for this dataset/lab:**

Ami

**Google Sheet:**
https://docs.google.com/spreadsheets/d/1E0w8UUVYyFIMkVLVZGPsyZKZK9-d18De/edit#gid=551291959

**Publication:**
Howden SE, Vanslambrouck JM, Wilson SB, Tan KS et al. Reporter-based fate mapping in human kidney organoids confirms nephron lineage relationships and reveals synchronous nephron formation. EMBO Rep 2019 Apr;20(4). PMID: 30858339

![image](https://user-images.githubusercontent.com/25744951/124604643-0ad82c80-de63-11eb-8c1b-8a8ac9a71f47.png)

**Acceptance criteria for the task:**

- [x] curate metadata
- [x] add ontologies
- [x] validate with graph validator
- [x] secondary review
- [ ] validate in ingest staging
- [ ] upload to ingest production

**Comment by ESapenaVentura** [comment]
Overall looks good, I just have a couple comments:

- On the project - Funders tab, I corrected a couple of funders
- On cell line and organoid tabs, I changed the cell line IDs to not contain hyphens, as it breaks some tools (e.g. the ingest graph validator)
- On the analysis protocol, it's missing the analysis protocol type (We should have the proper ontologies before the next release, so I'd wait to get those), and it seems that you have introduced `read alignment||transcript counting` in computational method, but I think it should be on derivation process?


I also ran the tests for ingest-graph-validator on the EC2 and seems alright!
